### ISSUE TITLE
current working directory changed on package load

### ISSUE THREAD

**Issue by r2evans** [emacs-ess/ESS#350]
In R, if I change the directory (`setwd`), I typically `M-return` (`ess-dirs`?) to set ESS's understanding of the working directory. However, when I close R and restart it, the working directory is now its parent directory. (If it makes a difference, the working directory for the R process is different from the directory of the package source code.)

This is all fine:

``` r
> getwd()
[1] "c:/Users/r2/parentdir/dir1"
# this is auto-inserted after `C-c C-t l`
> devtools::load_all('c:/Users/r2/Projects/Ford/adsRaceRS')
Loading adsRaceRS
> getwd()
[1] "c:/Users/r2/parentdir/dir1"
> q("no")
Process R finished at Mon Jul 11 11:24:48 2016
```

Then I restart R:

``` r
R version 3.2.5 (2016-04-14) -- "Very, Very Secure Dishes"
# ...snip...
> getwd()
[1] "c:/Users/r2/parentdir"
```

(The behavior is not seen if I manually type in `devtools::load_all(...)`, only when done from ESS.)

Currently using `ess-20160711.444` on emacs-24.5.1.

**Comment by r2evans** [comment]
Appears to no longer be a problem:

```
ess-version: 16.04 [elpa: 20160913.853] (loaded from c:/Users/r2/.emacs.d/elpa/ess-20160913.853/)
GNU Emacs 25.1.1 (x86_64-w64-mingw32) of 2016-09-17
```

**Comment by r2evans** [comment]
This is still an issue with emacs-24.5.1 (and there are performance issues with emacs-25 that I can't work around).

**Comment by jabranham** [comment]
Closing as a duplicate of #468 

I'll also note that this is unlikely to get attention since it's fixed in Emacs 25.1 and later. We plan on dropping support for Emacs < 25.1 after releasing ESS 18.09.
